src/config/backpack/operations/create.php

```diff
@@ -21,6 +21,9 @@
     // when the page loads, put the cursor on the first input?
     'autoFocusOnFirstField' => true,
 
+    // when enabled, the create button will open a modal form instead of the create page
+    'createButtonWithModalForm' => true,
```

Comment:
Wait... when did this the modalForm become the default?!?! Hell no!

```suggestion
    'createButtonWithModalForm' => false,
```